pyproject.toml

```diff
@@ -15,7 +15,7 @@ keywords = [
 license = "BSD-3-Clause"
 maintainers = [ {name = "The Scikit-HEP admins", email = "scikit-hep-admins@googlegroups.com"} ]
 authors = [ {name = "Jim Pivarski, Henry Schreiner, Eduardo Rodrigues", email = "eduardo.rodrigues@cern.ch"} ]
-requires-python = ">=3.6"
+requires-python = ">=3.7"
 dependencies = [
   'importlib-metadata>=0.22; python_version < "3.8"',
   "numpy>=1.13.3",
```

Comment:
I think this is now higher; the first version to support 3.7 was newer than 1.13.3.

Reply:
It's `1.14.5`.